gem/settings/base.py

```diff
@@ -37,31 +37,31 @@
 # Authentication Service Tokens
 USE_OIDC_AUTHENTICATION = environ.get(
     'USE_OIDC_AUTHENTICATION', '') == 'true'
-OIDC_RP_CLIENT_ID = "unused"  # Some constructors require that this be set.
-OIDC_RP_CLIENT_SECRET = "unused"  # some constructors require that this be set.
+OIDC_RP_CLIENT_ID = "118274"  # Some constructors require that this be set.
```

Comment:
the client ID and SECRET does not need to be here, it is set in the django-admin. In future, whern these are uploaded we need to change them and create new ones since github is public